packages/flutter_tools/lib/src/debug_adapters/flutter_adapter.dart

```diff
@@ -65,6 +65,14 @@ class FlutterDebugAdapter extends FlutterBaseDebugAdapter {
     'app.exposeUrl',
   };
 
+  /// A list of events from `flutter run --machine` that should be forwarded to the client.
+  final Set<String> _eventsToForwardToClient = <String>{
```

Comment:
Can this be const?

```suggestion
  static const Set<String> _eventsToForwardToClient = <String>{
```

Reply:
Good call. I pushed the same for `_requestsToForwardToClient` too. Thanks!